src/libasr/utils.h

```diff
@@ -37,6 +37,7 @@ struct CompilerOptions {
     bool implicit_typing = false;
     bool implicit_interface = false;
     std::string target = "";
+    std::string import_path = "";
```

Comment:
This will probably later be `std::vector<std::string>`, but for now that's fine.